tests/createServer.test.js

```diff
@@ -1,206 +1,73 @@
+'use strict';
 /* eslint-disable max-len */
 
-const { Server, get } = require('http');
+const http = require('http');
+const { convertToCase } = require('../src/convertToCase');
 
-function request(url = '/') {
-  return new Promise((resolve, reject) => {
-    get('http://localhost:5701' + url, (res) => {
-      let body = '';
+const createServer = () => {
+  const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
```

Comment:
The variable `availableCases` should be declared as a constant outside the `createServer` function, as it doesn't change throughout the function execution.